src/HotChocolate/Core/test/Types.Json.Tests/FromJsonDescriptorTests.cs

```diff
@@ -418,6 +446,12 @@ public class QueryNullProp
         [GraphQLType("Foo")]
         public JsonElement GetFoo() => JsonDocument.Parse(@"{ ""bar"": null }").RootElement;
     }
+     
+    public class QueryWithAlias
+    {
+        [GraphQLType("Foo")]
```

Comment:
This seems to be wrong... do you expect to access a different field with an alias on the document? An alias just is allowed to have an effect for the response where it changes the response name, but it always has to deliver the value from the property 'bar' in the json document.